src/overlays/actors/ovl_Bg_Ikninside/z_bg_ikninside.h

```diff
@@ -5,14 +5,16 @@
 
 struct BgIkninside;
 
+#define DMIKNINSIDE_GET_SWITCH(thisx) ((thisx)->params & 0xFE00)
```

Comment:
```suggestion
#define DMIKNINSIDE_GET_SWITCH(thisx) (((thisx)->params & 0xFE00) >> 9)
```
the `>> 9` should be included in the macro.